net-mail/vpopmail/vpopmail-5.4.33-r4.ebuild

```diff
@@ -0,0 +1,300 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools eutils fixheadtails qmail user
+
+HOMEPAGE="http://www.inter7.com/index.php?page=vpopmail"
+DESCRIPTION="Collection of programs to manage virtual email on Qmail servers"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE="clearpasswd ipalias maildrop mysql postgres spamassassin"
+REQUIRED_USE="mysql? ( !postgres )"
+
+DEPEND="virtual/qmail
+	maildrop? ( mail-filter/maildrop )
+	mysql? ( dev-db/mysql-connector-c:0= )
+	postgres? ( dev-db/postgresql:=[server] )
+	spamassassin? ( mail-filter/spamassassin )"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-5.4.9-access.violation.patch
+	"${FILESDIR}"/${PN}-lazy.patch
+	"${FILESDIR}"/${PN}-vpgsql.patch
+	"${FILESDIR}"/${PN}-double-free.patch
+	"${FILESDIR}"/${PN}-5.4.33-vdelivermail-add-static.patch
+	"${FILESDIR}"/${PN}-5.4.33-fix-those-vfork-instances-that-do-more-than-exec.patch
+	"${FILESDIR}"/${PN}-5.4.33-remove-unneeded-forward-declaration.patch
+	"${FILESDIR}"/${PN}-5.4.33-clean-up-calling-maildrop.patch
+	"${FILESDIR}"/${PN}-5.4.33-fix-S-tag-in-case-spamassassin-changed-the-file-size.patch
+	"${FILESDIR}"/${PN}-5.4.33-strncat.patch
+)
+DOCS=(
+	ChangeLog
+	doc/AUTHORS
+	doc/FAQ
+	doc/INSTALL
+	doc/README.activedirectory
+	doc/README.authvchkpw
+	doc/README.filelocking
+	doc/README.ipaliasdomains
+	doc/README.ldap
+	doc/README.maildrop
+	doc/README.mysql
+	doc/README.onchange
+	doc/README.oracle
+	doc/README.pgsql
+	doc/README.qmail-default
+	doc/README.quotas
+	doc/README.roamingusers
+	doc/README.spamassassin
+	doc/README.sybase
+	doc/README.vdelivermail
+	doc/README.vlimits
+	doc/README.vpopmaild
+	doc/README.vqmaillocal
+	doc/UPGRADE
+)
+HTML_DOCS=(
+	doc/doc_html/vpopmail5.abw
+	doc/doc_html/vpopmailapi.png
+	doc/doc_html/vpopmail.html
+	doc/man_html/clearopensmtp.html
+	doc/man_html/index.html
+	doc/man_html/vaddaliasdomain.html
+	doc/man_html/vadddomain.html
+	doc/man_html/vadduser.html
+	doc/man_html/vchkpw.html
+	doc/man_html/vconvert.html
+	doc/man_html/vdeldomain.html
+	doc/man_html/vdelivermail.html
+	doc/man_html/vdeluser.html
+	doc/man_html/vpasswd.html
+	doc/man_html/vpopbull.html
+	doc/man_html/vsetuserquota.html
+)
+
+# This makes sure the variable is set, and that it isn't null.
+VPOP_DEFAULT_HOME="/var/vpopmail"
+
+vpopmail_set_homedir() {
+	VPOP_HOME=$(egethome vpopmail)
```

Comment:
With acct-* method, this should probably be pre-defined, but Im not sure. Will this cause problems?

Reply:
The idea is that the user can set the home directory to a custom place. I tend to leave it as is for the moment and maybe drop that in the next iteration and just fix it to /var/vpopmail or whereever.